blivet/formats/__init__.py

```diff
@@ -367,6 +367,15 @@ def create(self, **kwargs):
         """
         log_method_call(self, device=self.device,
                         type=self.type, status=self.status)
+        self._preCreate(**kwargs)
+        try:
+            self._create(**kwargs)
+        except Exception:
+            raise
```

Comment:
I think the try-except doesn't make sense here. Maybe it had ``finally:`` or ``else:`` where it was taken from (?), but in the form it has here it doesn't make a difference to a plain ``self._create(**kwargs)`` call.